src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  const { content } = selectedTab;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                data-cy="Tab"
+                key={id}
+                className={cn({
+                  'is-active': selectedTab === tab,
+                })}
+              >
+                <a
+                  href={`#${id}`}
+                  data-cy="TabLink"
+                  onClick={() => onTabSelected(tab)}
```

Comment:
Consider checking here if the tab is clicked to not invoke the `onTabSelected` for the same tab